solutions/core-landing-zone/lz-folder/services-infrastructure/dns-project/dns.yaml

```diff
@@ -19,11 +19,11 @@
 apiVersion: dns.cnrm.cloud.google.com/v1beta1
 kind: DNSManagedZone
 metadata:
-  name: dns-project-id-standard-core-public-dns # kpt-set: ${dns-project-id}-standard-core-public-dns
+  name: core-dns-project-id-standard-core-public-dns # kpt-set: ${core-dns-project-id}-standard-core-public-dns
```

Comment:
changing this resource metadata.name will destroy the existing dns zone and create a new one ? right ?

Reply:
I don't see anything that would trigger a re-deployment. The resource ID is remaining intact.

```yaml
spec:
  description: "standard core public dns zone"
  resourceID: standard-core-public-dns
```

Reply:
no, the rendered metadata.name does not change...it's only the setter name.

```yaml
# setters.yaml
...
dns-project-id: my-dns-project
# renamed to 
core-dns-project-id: my-dns-project
```
```yaml
# rendered dns.yaml, the name remains unchanged
...
metadata:
  name: my-dns-project-standard-core-public-dns # kpt-set: ${core-dns-project-id}-standard-core-public-dns
```